test/run.js

```diff
@@ -141,6 +141,12 @@ if (semver.gte(process.version, "4.0.0")) {
   ]);
 }
 
+enqueue("mocha", [
+  "--harmony",
+  "--reporter", "spec",
+  "./test/frozen-intrinsics.js",
+]);
```

Comment:
@kumavis Nice, this `enqeue` will run the tests you added in an isolated child process, so the `Object.freeze` stuff doesn't leak into other tests.